service/src/main.rs

```diff
@@ -771,7 +746,7 @@ fn bootstrap_funds_from_alice(
 
 	if funding_amount > alice_free {
 		println!(
-			"funding amount is to high: please change MIN_FUND_INCREASE_FACTOR ({:?})",
+			"funding amount is to high: please change EXISTENTIAL_DEPOSIT_FACTOR_FOR_INIT_FUNDS ({:?})",
```

Comment:
Can't we just name it `ED_MULTIPLIER`?

Reply:
ED isn't very descriptive. As this is a variable that needs to be set for every node, i.e. for pretty much every use case, I'd prefer to use names that don't require much insider knowledge.

Sidenote: I didn't change the variable name, I only adapted the print message to match the variable name.

Reply:
I just think that the current name is equally cryptic, but then let's leave it.